backend/clickhouse/logs.go

```diff
@@ -223,10 +221,19 @@ func (client *Client) ReadSessionLogs(ctx context.Context, projectID int, params
 
 	for rows.Next() {
 		var result struct {
-			Timestamp    time.Time
-			UUID         string
-			SeverityText string
-			Body         string
+			Timestamp       time.Time
+			UUID            string
+			SeverityText    string
+			Body            string
+			LogAttributes   map[string]string
```

Comment:
i would expect fetching `LogAttributes` rather than just `Body` / a few of the main columns to be quite a bit slower, so i'm curious how this changes things

Reply:
Do we want the user to expand to see all the attributes? If not I think these changes can be ignored (will sync with @julian-highlight)

![Screenshot 2024-05-31 at 11 42 35 AM](https://github.com/highlight/highlight/assets/17744174/ea56ed56-d947-4464-9729-509e0db2d94a)